Source/Bogus/Hashids.cs

```diff
@@ -39,9 +39,6 @@ public Hashids() : this(string.Empty, 0, DEFAULT_ALPHABET, DEFAULT_SEPS)
       /// <summary>
```

Comment:
I'd try to avoid making changes to `Hashids.cs` when possible. This file is a copy/paste from the `/Source/hashids` submodule with "a few" changes to the original source. We should avoid changes to this file as much as possible to maintain closeness to the original source in case of the unlikely event we need to update this file again.

Ultimately, though, this submodule will be deprecated in the future.

Reply:
reverted